sklearn/linear_model/_perceptron.py

```diff
@@ -130,14 +130,22 @@ class Perceptron(BaseSGDClassifier):
         Number of weight updates performed during training.
         Same as ``(n_iter_ * n_samples)``.
 
+    See Also
+    --------
+    sklearn.linear_model.SGDClassifier : Linear classifiers 
```

Comment:
This is the cause of the linting failure:

```suggestion
    sklearn.linear_model.SGDClassifier : Linear classifiers
```

Reply:
We need a final full stop as well.

Reply:
No full stop needed, I did not see the line was continuing. My bad :)